Makefile

```diff
@@ -413,6 +413,7 @@ cinder_cleanup: ## deletes the operator, but does not cleanup the service resour
 
 .PHONY: cinder_deploy_prep
 cinder_deploy_prep: export KIND=Cinder
+cinder_deploy_prep: export IMAGE="${CINDER_IMG}"
```

Comment:
the IMAGE is usually the one to be used by the service (api/scheduler/volume). with cinder having multiple nested services, this is right now not supported here to set. it works as it kustomize won't fail with setting this here, but right now its to wrong image and won't have any affect and uses the images set in the sample cr. maybe we just set it to a string which shows that it does not result in a used setting?

Reply:
That sounds like a good idea, I'll update the patch.

Reply:
Yes, good point.  Elsewhere for other operators we set it to something to indicate that it is not used: https://github.com/openstack-k8s-operators/install_yamls/blob/259ac56930a128ea9ee99727b47cb99c5987b422/Makefile#L183